clients/src/main/java/org/apache/kafka/common/errors/GroupAuthorizationException.java

```diff
@@ -19,11 +19,21 @@
 public class GroupAuthorizationException extends AuthorizationException {
     private final String groupId;
 
-    public GroupAuthorizationException(String groupId) {
-        super("Not authorized to access group: " + groupId);
+    public GroupAuthorizationException(String message, String groupId) {
+        super(message);
         this.groupId = groupId;
     }
 
+    public GroupAuthorizationException(String groupId) {
```

Comment:
I think any constructor that takes just a String and does not take a message is a bit too error prone. We could use a factory method perhaps.